pepper-apis/src/main/resources/db-changes/schema/DDP-3709-file-upload.xml

```diff
@@ -23,16 +23,16 @@
             <column name="file_upload_guid" type="varchar(20)">
                 <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_guid_uk"/>
             </column>
-            <column name="blob_name" type="varchar(300)">
+            <column name="blob_name" type="varchar(1024)">
```

Comment:
According to [docs](https://cloud.google.com/storage/docs/naming-objects), blob object names can be up to 1024 bytes of UTF-8, which roughly translates to at most 1024 ASCII. So bumping this up.